src/components/welcome-page/Welcome.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../styles/variables.scss";
+
+.main-content {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    font-size: $linkedSmallFont;
+    color: $linkedDarkBlue;
+    background-color: $linkedLightBlue;
+    background: linear-gradient(0deg, rgba(40, 103, 178, 0.28895308123249297) 0%, rgba(255, 255, 255, 1) 71%);
+
+    .wrapper {
+        &__header {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            span {
+                font-size: 1.75em;
+                font-weight: 600;
+            }
+            &--icon {
+                font-size: 2em;
+            }
+        }
+        h4 {
+            padding: 0 50px;
+            font-size: 1.1em;
+            font-weight: normal;
+            letter-spacing: 1px;
+        }
+        &__btn {
```

Comment:
btns are bounded tightly to welcome page. That's why I'd like to write first components using storybook. Storybook helps us to write components which looks always the same regardless where we use it. This tight coupling is making buttons improsibble to use in other place in the app